roles/ec2_instance_create/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+- name: Run 'ec2_instance_create' role
+  module_defaults:
+    group/aws: "{{ aws_setup_credentials__output }}"
+  block:
+    - name: Create EC2 instance with provided configuration
+      amazon.aws.ec2_instance:
+        region: "{{ ec2_instance_create_aws_region }}"
```

Comment:
What if an instance with this name already exist?

Reply:
`ec2_instance_create_instance_name` param can be used by users to provide a name of their choice.

Reply:
added a check, playbook will fail if instance with same name exists in the region.